lib/wallet/account.js

```diff
@@ -508,7 +516,8 @@ class Account {
 
   /**
    * Save addresses to path map.
-   * @param {WalletKey[]} rings
+   * @param {Batch} b
+   * @param {WalletKey[]} ring
```

Comment:
are these actually arrays ? same with `Path[]` below

Reply:
Shouldn't be an array, my bad.